src/brogue/Monsters.c

```diff
@@ -387,7 +387,7 @@ void setPlayerDisplayChar() {
     if (rogue.easyMode) {
         player.info.displayChar = '&';
```

Comment:
I think this used to be `G_DEMON` too? 

```suggestion
        player.info.displayChar = G_DEMON;
```

Reply:
Done. I hadn't realized there was an icon for the player in easy mode. `G_DEMON` seems to have been unused even before the recent changes.